src/aggregate/aggregate_exec.c

```diff
@@ -1098,9 +1098,14 @@ static int execCommandCommon(RedisModuleCtx *ctx, RedisModuleString **argv, int
   return REDISMODULE_OK;
 
 error:
+  // Update global query errors statistics before freeing the request.
+  // Both SA and internal are considered shards.
```

Comment:
What's the difference between SA and internal? we don't get here if we have more than 1 shard